src/react/components/window/Popup.jsx

```diff
@@ -86,8 +87,17 @@ export const Popup = ThemeConsumer(( {title = '', children, onClose, bringToTop,
             return handleUnmounting;
         }
         // center after content has been rendered
+        let placement = options.placement;
+        // Open on bottom/top on mobile to prevent popup overflowing the screen
+        if (Oskari.util.isMobile()) {
+            if (placement && placement.indexOf('bottom') > -1) {
+                placement = PLACEMENTS.BOTTOM;
+            } else {
```

Comment:
This will move all popups that would open at center to top (if they are not configured to open on bottom). But lets see how it works.